### PR TITLE
Rename widget type wt_5 to slider

### PR DESCRIPTION
**Pull request by AaronVanGeffen**
Following up from #1610. I have changed the non-slider use of `wt_5`/`slider` to use `wt_6`, which does not change the appearance or behaviour of the widgets in question. Could do with another test, though.

### CODE REVIEW

**Comment by LeftofZen** [comment]
oh nice, I'll test this in a few mins, any idea what wt_6 is then?

File: src/OpenLoco/Windows/ObjectSelectionWindow.cpp [hunk 1]
```diff
@@ -88,7 +88,7 @@ namespace OpenLoco::Ui::Windows::ObjectSelectionWindow
         makeWidget({ 1, 1 }, { 598, 13 }, WidgetType::caption_25, WindowColour::primary, StringIds::title_object_selection),
         makeWidget({ 585, 2 }, { 13, 13 }, WidgetType::buttonWithImage, WindowColour::primary, ImageIds::close_button, StringIds::tooltip_close_window),
         makeWidget({ 0, 65 }, { 600, 333 }, WidgetType::panel, WindowColour::secondary),
-        makeWidget({ 3, 15 }, { 589, 50 }, WidgetType::wt_5, WindowColour::secondary),
+        makeWidget({ 3, 15 }, { 589, 50 }, WidgetType::slider, WindowColour::secondary),
```
Comment (LeftofZen) [inline src/OpenLoco/Windows/ObjectSelectionWindow.cpp:91]: I think this should now be wt_6?

Comment (AaronVanGeffen) [reply src/OpenLoco/Windows/ObjectSelectionWindow.cpp:91]: You're right! It is just used as a frame/canvas in this window.

Comment (AaronVanGeffen) [reply src/OpenLoco/Windows/ObjectSelectionWindow.cpp:91]:
```suggestion
        makeWidget({ 3, 15 }, { 589, 50 }, WidgetType::wt_6, WindowColour::secondary),
```

**Review by LeftofZen — APPROVED** [review]
lgtm, wt_6 could be verticalScrollbar?

**Comment by AaronVanGeffen** [comment]
> lgtm, wt_6 could be verticalScrollbar?

I think you're looking at a different widget. The scrollbars are part of the `scrollview` widget type.

**Comment by LeftofZen** [comment]
> > lgtm, wt_6 could be verticalScrollbar?
> 
> I think you're looking at a different widget. The scrollbars are part of the `scrollview` widget type.

oh right, yeah I didn't look too deeply, the scrollbar was the only thing i saw in common so I assumed it was that

**Pull request merged**